### PR TITLE
Updating with latest from DefinitelyTyped

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[375441a](https://github.com/OfficeDev/office-js-docs-reference/commits/375441a36a7e00ed3a0378c637cebff956a2a5ee)_: 

:clock10: **Pending**: waiting for processors ([13](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 13 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

File: docs/docs-ref-autogen/excel/excel/excel.workbookrangeareas.yml [hunk 1]
```diff
@@ -41,8 +41,8 @@ properties:
     package: excel!
     fullName: areas
     summary: >-
-      Returns the `RangeAreasCollection` object. Each `RangeAreas` object in the
-      collection represents one or more rectangle ranges in one worksheet.
+      Returns the `RangeAreasCollection` object. Each `RangeAreas` in the
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/excel/excel/excel.workbookrangeareas.yml:44]: Can they revert this change upstream? I think Ricky and I agreed that the original read better than the new version.

Comment (AlexJerabek) [reply docs/docs-ref-autogen/excel/excel/excel.workbookrangeareas.yml:44]: Yeah, I've got that in my change list. I'll fix it in DT next time.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Comment by o365devx** [comment]
Docs Build status updates of commit _[375441a](https://github.com/OfficeDev/office-js-docs-reference/commits/375441a36a7e00ed3a0378c637cebff956a2a5ee)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel/excel.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.comment.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.comment.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.commentreply.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.commentreply.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.errorcodes.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.errorcodes.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.eventtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.eventtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangecollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangecollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangefilldata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangefilldata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangefillloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangefillloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangefillupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangefillupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangeloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangeloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.rangeupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.rangeupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskcollectiondata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskcollectiondata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskcollectionupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskcollectionupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordcollectiondata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordcollectiondata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordcollectionupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordcollectionupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecorddata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecorddata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskhistoryrecordupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.taskloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.taskloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.textconditionalformatdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.textconditionalformatdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.textconditionalformatloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.textconditionalformatloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.textconditionalformatupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.textconditionalformatupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.topbottomconditionalformatdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.topbottomconditionalformatdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.topbottomconditionalformatloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.topbottomconditionalformatloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.interfaces.topbottomconditionalformatupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.interfaces.topbottomconditionalformatupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.pivottabledategroupby.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.pivottabledategroupby.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.rangefill.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.rangefill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.task.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.task.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.taskchanges.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.taskchanges.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.taskcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.taskcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.taskhistoryrecord.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.taskhistoryrecord.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.taskhistoryrecordcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.taskhistoryrecordcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.taskhistoryrecordtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.taskhistoryrecordtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.textconditionalformat.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.textconditionalformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.topbottomconditionalformat.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.topbottomconditionalformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.user.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.user.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.workbookrangeareas.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.workbookrangeareas.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.errorcodes.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel_1_1/excel/excel.errorcodes.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.interfaces.rangedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel_1_1/excel/excel.interfaces.rangedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.interfaces.rangeloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel_1_1/excel/excel.interfaces.rangeloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.interfaces.rangeupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel_1_1/excel/excel.interfaces.rangeupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_10/excel/excel.errorcodes.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DTUpdates-11-17/docs/docs-ref-autogen/excel_1_10/excel/excel.errorcodes.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C17%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202011171932568052-835%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qQ8EWl1vRN5hfvThbt5mUk8cTG7dqQ1LPJqsHcowDyY%3D&st=2020-11-17T19%3A42%3A18Z&se=2020-12-18T19%3A47%3A18Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0e1871f2-88cb-c16b-6d81-dc80e4f66cef?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
